tasks/eva_3372/src/main/groovy/eva3372/CollectSplitsAndMerges.groovy

```diff
@@ -63,18 +63,27 @@ class CollectSplitsAndMerges {
         def svesGroupedByRsId = sves.groupBy { it.clusteredVariantAccession }
         def rsIds = svesGroupedByRsId.keySet()
         def existingRecords = devEnv.mongoTemplate.find(
-                query(where("_id").in(rsIds.collect { "${assembly}#${it}" })), CveHashesWithRsId.class)
+                query(where("_id").in(rsIds.collect { "${assembly}#${it}".toString() })), CveHashesWithRsId.class)
```

Comment:
It's apparently important to convert this from a Groovy string for this query to work properly, even though all the inserts etc. work just fine! The more you know...